src/main/scala/edu/ie3/simona/sim/setup/SimonaStandaloneSetup.scala

```diff
@@ -66,6 +66,7 @@ class SimonaStandaloneSetup(
     resultFileHierarchy: ResultFileHierarchy,
     runtimeEventQueue: Option[LinkedBlockingQueue[RuntimeEvent]] = None,
     override val args: Array[String],
+    override val logOutputDir: String,
```

Comment:
You could also add `override val logOutputDir: String = resultFileHierarchy.logOutputDir` to the class body instead. That way, we'll have one less variable within the class parameters.